### ISSUE TITLE
Tabbing issue on left nav - a11y

### ISSUE THREAD

**Issue by tay-aitken** [carbon-design-system/design-system-website#208]
After tabbing past "Components" the focus disappears, it's technically moving through the sub-menu items of Components but it should have moved instead to the next node of "Add-Ons". Forces you to tab through all of the components, big pain.

Originally brought up in #167 (number 5 in the list)
